src/main/java/com/example/amazonreviewsapp/mapper/UserMapper.java

```diff
@@ -0,0 +1,29 @@
+package com.example.amazonreviewsapp.mapper;
+
+import com.example.amazonreviewsapp.model.dto.UserDto;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapper {
+
+    public UserDto getUserDtoFromLine(String[] line) {
+        UserDto userDto = new UserDto();
+        userDto.setProductId(line[1]);
```

Comment:
```suggestion
        userDto.setProductId(line[PRODUCT_ID_POSITION]);
```
старайся уникати magic numbers